packages/components/tabs/tabs.js

```diff
@@ -1,313 +1,309 @@
-// import "../../vendor/polyfills/Function/prototype/bind";
```

Comment:
@wa-rren-dev This diff is super noisy - could you summarise what things you have changed here?